elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -119,7 +216,8 @@ public ElideResponse get(
                     mapper,
                     auditLogger,
                     queryParams,
-                    securityMode);
+                    securityMode,
```

Comment:
Yes for back-compat. Until we fully remove `securityMode`, this unfortunately needs to remain in place :-1: